openmetadata-ui/src/main/resources/ui/src/components/TeamDetails/TeamHierarchy.tsx

```diff
@@ -132,6 +134,34 @@ const TeamHierarchy: FC<TeamHierarchyProps> = ({
     }
   };
 
+  const tableExpandableIconData = useCallback(
+    ({ expanded, onExpand, expandable, record }: TableExpandableDataProps) =>
+      expandable ? (
+        <div
+          draggable
+          className="expand-cell-icon-container"
+          data-testid="expand-table-row"
+          onClick={(e) =>
+            onExpand(
+              record,
+              e as unknown as React.MouseEvent<HTMLElement, MouseEvent>
+            )
+          }>
+          <SVGIcons className="drag-icon" draggable="true" icon={Icons.DRAG} />
+          <SVGIcons
+            className="expand-icon"
+            icon={expanded ? Icons.ARROW_DOWN_LIGHT : Icons.ARROW_RIGHT_LIGHT}
+          />
+        </div>
+      ) : (
+        <>
+          <SVGIcons className="drag-icon" icon={Icons.DRAG} />
+          <div className="expand-cell-empty-icon-container" />
+        </>
+      ),
+    []
+  );
```

Comment:
Remove it, we have it at `tableUtils` use that one, with additional prop with drag-icon